Tests/OpenMP/atomicAdd/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+if (AMReX_GPU_BACKEND STREQUAL NONE)
```

Comment:
```suggestion
if (NOT AMReX_GPU_BACKEND STREQUAL NONE)
```

Reply:
Thanks! That's right. Now I understand. I wrote the correct expression here first and then copy-and-pasted to the other place. Copy-and-paste is a major source of my bugs.